src/editor/map/map_context.hpp

```diff
@@ -107,7 +107,7 @@ class map_context : public display_context
 
 	void new_area(const std::set<map_location>& area) {
 		tod_manager_->add_time_area("", area, config());
-		active_area_ = tod_manager_->get_area_ids().size() -1;
+		active_area_ = static_cast<int>(tod_manager_->get_area_ids().size()) - 1;
```

Comment:
Might be better to make `active_area_` size_t?

Reply:
If `tod_manager_->get_area_ids().size()` returns 0, then `active_area_` would end up as -1.

Reply:
Agh. Bad design but out of scope to fix here. >_<